content/about/tob.md

```diff
@@ -62,29 +64,25 @@ CNCF containerd project and has been active in the OCI since its founding
 in 2015.
 {{< /card >}}
 
-{{< card img="/img/tob/Wei-Fu.jpeg" name="Wei Fu" termdates="2020-01-29 → 2022-01-29" company="Alibaba" github="https://github.com/fuweid">}}
-Maintainer of the CNCF's containerd project.
-{{< /card >}}
-
-{{< card name="Jon Johnson" termdates="2020-01-29 → 2022-01-29" company="Google" github="https://github.com/jonjohnsonjr" >}}
+{{< card name="Jon Johnson" termdates="2022-01-29 → 2024-01-29" company="Google" github="https://github.com/jonjohnsonjr" >}}
 Software engineer at Google working on containers.
 {{< /card >}}
 
-{{< card img="/img/tob/Samuel-Karp.jpeg" name="Samuel Karp" termdates="2020-01-29 → 2022-01-29" company="Amazon Web Services" github="https://github.com/samuelkarp" twitter="https://twitter.com/samuelkarp" personal="https://samuelkarp.com" >}}
+{{< card img="/img/tob/Samuel-Karp.jpeg" name="Samuel Karp" termdates="2022-01-29 → 2024-01-29" company="Amazon Web Services" github="https://github.com/samuelkarp" twitter="https://twitter.com/samuelkarp" personal="https://samuelkarp.com" >}}
 Samuel Karp is a Senior Software Development Engineer working on containers at
 Amazon Web Services. Sam first contributed to Docker in 2015 and containerd in
 2017. Sam's current focus is on container runtimes and how the runtimes
 interact with orchestrators and registries. Sam has written code and designs
 for Amazon ECS and AWS Fargate and consulted for Amazon ECR and Amazon EKS.
```

Comment:
My company affiliation has changed (Google) and the description isn't super accurate anymore.  Do you prefer to get this merged first and then have TOB members update their own description or would you prefer to have it as a suggestion here?

Reply:
Either is fine with me. While I've got the PR open, feel free to send the change over to me.

Reply:
@samuelkarp send over your updated description since the affiliation change got included in my last update. Thanks!

Reply:
Updated. Thanks!